extension/llm/export/config/llm_config.py

```diff
@@ -227,6 +227,20 @@ def __post_init__(self):
             )
 
 
+@dataclass
+class SerializationConfig:
+    """
+    Configures properties relevant to the serialization process.
+
+    Attributes:
+        foundation_weights_file: configure the foundation weights of a model
+            to be placed in a separate file, external to the PTE. Pass the
```

Comment:
Are you expecting a lot of other fields to be here? Could we just add this a field in `ExportConfig` instead since it has to do with how we export into the pte?